grapher/structures.py

```diff
@@ -154,6 +154,8 @@ def get_source_files(diry: str) -> List[str]:
     return files
 
 def pkgToUnitKey(pkg: Dict) -> UnitKey:
+    if pkg['project_name'] is None:
```

Comment:
- add in `requirements.txt` `-e .`
- I think that `-e http://google.com` will work too (any URL that does not point to github/bitbucket/google code)